locales/messages.en.json

```diff
@@ -185,19 +191,23 @@
   "ProcessorArchitectureMissing": "The required environment variable %PROCESSOR_ARCHITECTURE% is missing.",
   "ProcessorArchitectureW6432Malformed": "Failed to parse %PROCESSOR_ARCHITEW6432% ({arch}) as a valid CPU architecture. Falling back to %PROCESSOR_ARCHITECTURE%.",
   "ProgramReturnedNonzeroExitCode": "{tool_name} failed with exit code: ({exit_code}).",
+  "PushingVendorFailed": "Pushing {vendor} to '{path}' failed.  Use --debug for more information.",
   "RemovingPackage": "Removing {action_index}/{count} {spec}",
-  "RestoredPackagesFromVendor": "Restored {count} package(s) from {vendor} in {elapsed}",
+  "ReplaceSecretsError": "'{error_msg}'",
```

Comment:
I don't know if I agree with registering a localizable string just to surrond with `'`. Probably should use raw format here.

Reply:
I could just add a statement before the error message. Something like "Replace secretes produced the following error: '{error_msg}'." Is that cool with you or would you prefer to just write unlocalized to stdout?

Reply:
Yeah, we should add context to the message.